addons/quickmount/functions/fnc_getInNearest.sqf

```diff
@@ -0,0 +1,50 @@
+/*
+ * Author: Kingsley
+ * Mount the given player in the nearest eligible vehicle based on their distance and direction to the vehicle
+ *
+ * Arguments:
+ * 0: Player unit <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [ace_player] call acex_quickmount_fnc_getInNearest;
+ *
+ * Public: No
+ */
+
+#include "script_component.hpp"
+
+params [["_unit", objNull, [objNull]]];
+
+if (!GVAR(Enabled)) exitWith {};
+if (isNull _unit || {vehicle _unit != _unit} || {!alive _unit} || {!isPlayer _unit}) exitWith {};
```

Comment:
I'd remove `isPlayer _unit`  check, or at least replace it with the corresponding function in `ACE_Common`, which is 100% remote control and switch unit compatible.

Also this function gets `_unit` as parameter, but what it does doesn't really make sense with any unit except the `ACE_player`. Maybe you could just replace `_unit` by `ACE_player` everywhere and ditch the argument.